pkgs/applications/audio/axoloti/default.nix

```diff
@@ -20,7 +21,7 @@ stdenv.mkDerivation rec {
     sha256 = "0lb5s8pkj80mqhsy47mmq0lqk34s2a2m3xagzihalvabwd0frhlj";
   };
 
-  buildInputs = [ makeWrapper unzip gcc-arm-embedded dfu-util-axoloti jdk ant libfaketime ];
+  buildInputs = [ makeWrapper unzip gcc-arm-embedded binutils-arm-embedded dfu-util-axoloti jdk ant libfaketime ];
```

Comment:
Some of these should be nativeBuildInputs but whatever, one problem at a time :)